dev/.env

```diff
@@ -15,7 +15,7 @@ EMR_WEIGHT_CONCEPT=
 CREATE_CUSTOMER_IF_NOT_EXIST=false
 ODOO_CUSTOMER_EXTERNAL_ID=comment
 ODOO_CUSTOMER_DOB_FIELD=
-ODOO_CUSTOMER_WEIGHT_FIELD=
+ODOO_CUSTOMER_WEIGHT_FIELD=x_customer_weight
```

Comment:
Are you using this anywhere, which I guess should have been transitively for the property commented on above?

Reply:
`ODOO_CUSTOMER_WEIGHT_FIELD` is not being used anywhere so maybe its better that I remove it.